docs/content/en/docs/implementing/tasks/python.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Working with Python runtime for Keptn tasks
```

Comment:
This file should be removed from this PR.  The text was stolen and incorporated into the implementing/tasks file in https://github.com/keptn/lifecycle-toolkit/pull/1521